### DISCUSSION TITLE
Todoism 设置FLASK_RUN_HOST 0.0.0.0  无法显示页面内容

### DISCUSSION THREAD

**Discussion by wggglggg** [greyli/helloflask#247] [Q&A]
小白刚学习这一章,  javascript把我难到了,  只学过一点点的javascript,  clone代码到本地后, 当作范本,  host是默认的127.0.0.1 也当场run 了一下看看范本渲染出来的是什么样子的, 又另建了一个项目文件夹 边看着范本边往这个自己项目的文件夹敲代码,  最基础的代码都搭好后, 先run 一下看看模样  ,  为了不与clone的ip有冲突, 把自己的项目改成FLASK_RUN_HOST  = 0.0.0.0,  发现自己的项目首页无法显示, 后来找问题,  我把clone的文件, 一个一个的替换自己手敲的代码,  所有的代码都替换完了, 首页还是无法显示 ,  突然想了 一下, 会不会是Ip的问题,  于是删除了0.0.0.0,  首页终于显示 出来 了,  自己手敲的代码文件都被 替换完了,  明天再重来.

**Comment by greyli** [comment]
Hello，「项目首页无法显示」时你访问的 URL 是什么？命令行的输出是什么？启动的命令是什么？麻烦提供一些详细的相关信息。

**Reply by wggglggg** [reply]
我去, 发错地方了.
昨天 源码用的127.0.0.1:5000 edge浏览器,  可以正常显示 项目的首页.
自己代码用的192.168.1.116:5000 edge浏览器, 发现192.168这台打不开网页, 显示白色背景,  于是把源码127.0.0.1那台也改成了FLASK_RUN_HOST = 0.0.0.0,  先停用我自己的代码服务器, 然后源码用192.168.1.116:5000,  网页也打不开了, 背景也白色.
重启电脑发现源码还是显示白色,  刚才直接把chrome 卸载掉, 再重启电脑 , 重装chrome, 现在可以了,  127.0.0.1和192.168.1.116都能打开首页了
